Doc/pages/scattering.rst

```diff
@@ -947,13 +951,33 @@ one is, etc.
 -  :ref:`param-output-files`
 -  :ref:`param-running-mode`
 
-.. _neutron-dynamic-total-structure-factor:
+.. _analysis-ndtsf:
 
 Neutron Dynamic Total Structure Factor
 ''''''''''''''''''''''''''''''''''''''
 
 -  available for trajectories only
 
+This is a specific combination of the Dynamic Coherent and the
```

Comment:
I think that this is not very clear. I will try to prepare an alternative text and then I let you decide what to write.